ansible/vars/csm_packages.yml

```
@@ -66,12 +67,14 @@ application_csm_sles_packages:
   - cray-uai-util
   - smart-mon
   - smartmontools
+  - cray-spire-dracut>=2.0.0
```

Same deal as the `spire-agent` comment. We could just specify the package name here with no version and it would have the same result, except in the case where this RPM is in Nexus but only for versions less than 2.0.0.

Again, no need to take this suggestion if this is the desired behavior.

```suggestion
  - cray-spire-dracut
```

---

I also had concerns here with it pulling the version from the COS repos I think this one could be changed but its not what I tested.